docs/docs/getting-started/providers/credentials-tutorial.mdx

```diff
@@ -122,13 +122,11 @@ export default NextAuth({
   Auth.js provides us a getSession, function to access the session data and status, to call from the `event.locals` variable. We can now just call it and add it to our `$page` store.
 
   ```ts title="route/+layout.server.ts"
-  import type { LayoutServerLoad } from './$types';
-  
-  export const load: LayoutServerLoad = async (event) => {
-    return {
-      session: await event.locals.getSession()
-    };
-  };
+  export async function load({ locals }) {
```

Comment:
How would you type this then?

Reply:
Sorry if I'm misunderstanding but SvelteKit generates the types automatically through 

```
+page.ts -> PageLoad
+page.server.ts -> PageServerLoad
+layout.ts -> LayoutLoad
+layout.server.ts -> LayoutServerLoad
```

Reply:
That's correct

Reply:
Yeah, I tried that, it didn't pick up the `.svelte-kit/*` autogenerated types, but maybe it's just an issue with my IDE then

Reply:
You need to have Svelte extension installed - in case it wasn't. Sometimes reloading the window can also fix those issues.